packages/notepadplusplus-compare.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,27 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $pluginsDir= Join-Path ${Env:ProgramFiles} "Notepad++\plugins" -Resolve
+  $toolDir = New-Item "C:\Program Files\Notepad++\plugins\ComparePlugin" -itemtype directory
```

Comment:
Let's use the `$pluginsdir` instead of a hard-coded path, like:
`$toolDir = New-Item "$pluginsDir\ComparePlugin" -itemtype directory`